src/dotnet/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen.Tests/TestApiComparer.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.SDK.ChangelogGen.Compare;
+using Azure.SDK.ChangelogGen.Report;
+
+namespace Azure.SDK.ChangelogGen.Tests
+{
+    [TestClass]
+    public class TestApiComparer
+    {
+        [TestMethod]
+        public void TestCompareApiFile()
+        {
+            string content1 = File.ReadAllText("apiFile1.cs.txt");
+            string content2 = File.ReadAllText("apiFile2.cs.txt");
+            ChangeLogResult r = new ChangeLogResult();
+            r.ApiChange = Program.CompareApi(content2, content1);
+            Release release = r.GenerateReleaseNote();
+            release.Version = "1.2.3";
+            release.ReleaseDate = "2030.3.3";
+
+            string baseline =
+@"## 1.2.3 (2030.3.3)
+
+### Breaking Changes in API
```

Comment:
These are not standard sections for an Azure SDK change log.

Reply:
Do you mean the title? or the content? If you are talking about the API diff list, will update the baseline after we get a conclusion of the discussion.

Reply:
Both; please see the [Azure SDK policy for Release Notes](https://azure.github.io/azure-sdk/policies_releasenotes.html).   While change logs can (and often should) have greater detail, they need to follow the overall structure and should follow the same guiding principles as the release notes.

Reply:
Also, I really think we need to make sure we use different terminology for real "breaking changes" (breaks off a GAed package baseline) and changes off a preview package baseline. Many users want a concise and noise-free way to discover real breaking changes that we sometimes must do. For majority of devs who don't use our previews calling intra-release changes "breaking" constitutes noise.

Reply:
updated per our conversation. thanks.